rim/views.py

```diff
@@ -84,6 +84,23 @@ def get_queryset(self):
         qset = qset.select_related('latest_checkout__location', 'equipment_type')
         return qset
 
+class ListClientView(ListView):
+    template_name = 'rim/client_list.html'
+    model = Client
+    form_class = ClientForm
+
+    def get_queryset(self):
+        queryset = super(ListClientView, self).get_queryset()
```

Comment:
No need for the call to `super()` if you're just redefining `queryset` anyway

Reply:
fixed